developer_setup/classic_ui_split.md

```diff
@@ -14,7 +14,7 @@ But if you do..
 1. `git clone git@github.com:YOU/manageiq-ui-classic`
 1. `cd manageiq-ui-classic`
 1. `git remote add upstream git@github.com:ManageIQ/manageiq-ui-classic.git`
-1. `ln -s ../../manageiq spec/`
+1. `ln -s ../manageiq spec/`
```

Comment:
@Fryguy @chessbyte it's weird .. but this is not working for me .. I need to use ```ln -s ../../manageiq spec/``` to create symlink with proper path .. maybe some difference between OSX and Fedora in ```ln``` ?